### DISCUSSION TITLE
Instances are not running and content is not loading.

### DISCUSSION THREAD

**Discussion by Pamilg8** [libre-tube/LibreTube#5580] [General]
### Steps to reproduce

1.I checked the servers and they are fine
2. Internet connection is good.
4. LibreTube does not load anything. Sometimes only thumbnails. Or Nothing is loading 

### Expected behavior

The application is like this from yesterday evening. To stabilize the operation of the application

### Actual behavior

Steps to reproduce 

### LibreTube version

0.21.1 Debug

### Android version

Android 13

### Other details

I came to another city in the evening. LibreTube is not working. The instance is close to me😢
It seems that Wi-Fi is blocking the internet for LibreTube

https://github.com/libre-tube/LibreTube/assets/149590337/b8ccc421-34da-43c3-8745-0d4b598f3165

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by VitoItalianGamer** [comment]
Have the same issue as well.

**Comment by akano12** [comment]
Try enabling HLS and see if it fix your issue (settings=> audio and video)

**Comment by Pamilg8** [comment]
> Try enabling HLS and see if it fix your issue (settings=> audio and video)

The problem is not only playing the video, but the minatures etc. But now it's fixed. Related content still not loading. In case of network changes, the application stops working for a while

**Comment by Bnyro** [comment]
You can also try to clear the app's cache.

**Reply by Pamilg8** [reply]
I cleared the data . I downloaded the app again. It doesn't fix

**Comment by Pamilg8** [comment]
Related content is already loading. But the thumbnails are often not loaded. Or the content doesn't load at all.
